src/components/AnalysisChart/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import {
+  Chart,
+  ChartBar,
+  ChartAxis,
+  ChartStack,
+  ChartVoronoiContainer,
+  ChartDonutThreshold,
+  ChartDonutUtilization,
+} from '@patternfly/react-charts';
+
+class AnalysisChart extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      barWidth: '20',
+    };
+  }
+
+  render() {
+    const { barWidth } = this.state;
+    const { chartType, usedSpace, barData, chartColor } = this.props;
+
+    const donut = (
+      <ChartDonutThreshold
+        ariaDesc="Storage capacity"
+        ariaTitle="Donut utilization chart with static threshold example"
+        constrainToVisibleArea
+        data={[{ x: 'Warning at 60%', y: 60 }, { x: 'Danger at 90%', y: 90 }]}
+        labels={({ datum }) => (datum.x ? datum.x : null)}
+      >
+        <ChartDonutUtilization
+          data={{ x: 'Storage capacity', y: usedSpace }}
+          labels={({ datum }) => (datum.x ? `${datum.x}: ${datum.y}%` : null)}
+          thresholds={[{ value: 60 }, { value: 90 }]}
+        />
+      </ChartDonutThreshold>
+    );
+    const bar = (
+      <Chart
+        ariaDesc="mock metdata"
+        ariaTitle="mock meta data"
```

Comment:
Is this a mock component or the real version?

Reply:
This is a chart component is generated with mock data

Reply:
Right, but why does the real code reference the `mock` module?